main.js

```diff
@@ -37,12 +39,39 @@ module.exports.setConfig = function () {
 };
 
 module.exports.init = function (el) {
-	return oCommentUtilities.initDomConstruct({
-		context: el,
-		classNamespace: 'o-comments',
-		eventNamespace: 'oComments',
-		module: module.exports
-	});
+	if (!el) {
+		el = document.body;
+	} else if (!(el instanceof HTMLElement)) {
+		el = document.querySelector(el);
+	}
+
+	let instances;
+	if (el.dataset.oComponent === 'o-comments') {
+		instances = [el];
+	} else {
+		instances = Array.from(el.querySelectorAll('[data-o-component="o-comments"]'));
```

Comment:
Oh, we can't start using `Array.from` without a breaking change because `o-comments` users would have to add a polyfill for [IE support](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#Polyfill).

[Here's a way around it](https://davidwalsh.name/nodelist-array):

```suggestion
		instances = [].slice.call(el.querySelectorAll('[data-o-component="o-comments"]'));
```